syntax = "proto3";
package wilson;

import "google/protobuf/timestamp.proto";

//Meta information about a project
message ProjectMeta {
    //Name of the project
    string name = 1;
    //Author of the project
    string author = 2;
    //Date of event
    google.protobuf.Timestamp date = 3;

    //Time value, from which the animation starts
    double startTime = 4;
    //Time value, where the animation ends
    double endTime = 5;
    //Playback speed in time units per second
    double speedRatio = 6;

    //additional description of the event
    string description = 7;
}
